.circleci/docker/install_conda.sh

```diff
@@ -30,8 +30,11 @@ function install_and_setup_conda() {
   conda update -y -n base conda
   conda install -y python=$PYTHON_VERSION
 
-  conda install -y numpy=1.18.5 pyyaml mkl-include setuptools cmake cffi typing \
-    tqdm coverage hypothesis dataclasses cython
+  conda install -y nomkl numpy=1.18.5 pyyaml setuptools cmake \
```

Comment:
ditto, we we still need nomkl?

Reply:
yes, to make sure that other packages don't install mkl.